frontend/endpoints/destinations.go

```diff
@@ -91,6 +93,12 @@ func GetDestinationTypes(c *gin.Context) {
 		})
 	}
 
+	potentialDestinations, err := findPotentialDestinations(c)
+	if err != nil {
+		return
+	}
+	fmt.Printf("Potential destinations: %v\n", potentialDestinations)
```

Comment:
The potential destinations should be included in the response in some way (removing this print before merging)